connectors-home/modules/ROOT/partials/acb-java-module-partial-edit.adoc

```diff
@@ -0,0 +1,17 @@
+//Used in the configure-java-module-acb file for Java Module configuration in ACB
+
+// tag::advanced-tab-attributes[]
+. Select the *Advanced* tab to configure the following attributes: 
++
+[%header%autowidth.spread]
+|===
+| UI Field | Description | Required
+| Output MIME type | MIME type of the payload this component outputs.| No
+| Parameters | Appends a key-value pair for a reader property to the value of the Output MIME attribute. Multiple key-value pairs are allowed.| No
+| Output encoding | Encoding of the payload this component outputs. | No
+| Target variable | Name of the variable that stores the operation's output. | No
+| Target value | Expression that evaluates the operation's output. The expression outcome is stored in the target variable. | No
+| Error Mappings a| Creates custom error types that you map to Mule error types. The mappings can help you further differentiate and group errors in your app. This attribute is not currently enabled in the configuration UI. Configure it in the XML editor. See xref:mule-runtime::mule-error-concept.adoc[]. | No
```

Comment:
Any way to combine the third and fourth sentences (and put a positive spin) (and use contraction)?